mysql_protocol.cc

```diff
@@ -188,14 +194,60 @@ bool Protocol::Authenticate() {
     // there should atleast be a \0 byte as password
     return false;
   }
-  const char *passwd = user + user_len + 1;
-  int passwd_len = Unpack(&passwd);  // length encoded
+  const char *authdata = user + user_len + 1;
+  int authdata_len = Unpack(&authdata);  // length encoded
+
+  // TODO(crystall): Add monitoring for authentication failures
+  LOG(WARNING) << "Authenticating user: " <<  user;
+
+  const char *slave_pwhash = FLAGS_ripple_server_password_hash.c_str();
+
+  if (strlen(slave_pwhash) > 0) {
+    // Authentication with mysql_native_password
+
+    if (authdata_len < SHA_DIGEST_LENGTH) {
+      LOG(WARNING) << "Received authdata with length "
```

Comment:
Let's reduce this LOG to 2 lines only. Our style has 80 character limit and it's better to fill closer to the limit without splitting into lines too much.

Reply:
Fixed